tests/new_unit_tests/Commands/Connection/ConnectionCommandTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ *
+ * Contains Pantheon\Terminus\UnitTests\Commands\Connection\ConnectionCommandTest
+ */
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+
+use ReflectionMethod;
+
+abstract class ConnectionCommandTest extends \PHPUnit_Framework_TestCase
```

Comment:
It would make sense to have in the contract a few scenarios out which err for simple reasons (e.g. no internet access, a given file is unreadable), but I don't think it would serve us to seek out edge cases in our feature tests, which we would use unit tests to more effectively assess.